pkg/executor/internal/mpp/local_mpp_coordinator.go

```diff
@@ -498,6 +498,7 @@ func (c *localMppCoordinator) cancelMppTasks() {
 func (c *localMppCoordinator) receiveResults(req *kv.MPPDispatchRequest, taskMeta *mpp.TaskMeta, bo *backoff.Backoffer) {
 	stream, err := c.sessionCtx.GetMPPClient().EstablishMPPConns(kv.EstablishMPPConnsParam{Ctx: bo.GetCtx(), Req: req, TaskMeta: taskMeta})
 	if err != nil {
+		stream.Close()
```

Comment:
How about move L511 after L499?

Reply:
Golang lint error: `pkg/executor/internal/mpp/local_mpp_coordinator.go:500:2: should check returned error before deferring stream.Close() (SA5001)`